Assets/HoloToolkit/Sharing/Scripts/SDK/SystemRole.cs

```diff
@@ -0,0 +1,18 @@
+/* ----------------------------------------------------------------------------
+ * This file was automatically generated by SWIG (http://www.swig.org).
+ * Version 3.0.2
+ *
+ * Do not make changes to this file unless you know what you are doing--modify
+ * the SWIG interface file instead.
+ * ----------------------------------------------------------------------------- */
+
+namespace HoloToolkit.Sharing {
+
+public enum SystemRole {
+  SessionDiscoveryServerRole = 0,
```

Comment:
This is different than "HoloToolkit.Sharing.ClientRole" which is used in the client to specify Primary Client or Secondary Client from the user perspective. That remained unchanged with Primary Client being default.
This enum is used by the sharing framework to respond to a Network Discovery broadcast and I assume there is an internal mapping for the Client roles.